vision_agent/agent/vision_agent.py

```diff
@@ -366,6 +366,19 @@ def _handle_viz_tools(
     return image_to_data
 
 
+def sample_n_evenly_spaced(lst: Sequence, n: int) -> Sequence:
```

Comment:
is the `lst` here the list of outputs for all the frames ?

Reply:
yeah it would be the frames, but can generally be any list